lib/src/openApiToTypescript.ts

```diff
@@ -294,8 +294,7 @@ TsConversionArgs): ts.Node | TypeDefinitionObject | string => {
                 throw new Error("Name is required to convert an object schema to a type reference");
             }
 
-            const base = t.type(inheritedMeta.name, doWrapReadOnly(objectType));
-            if (!isPartial) return base;
+            if (!isPartial) return t.type(inheritedMeta.name, doWrapReadOnly(objectType));
```

Comment:
This adjustment was necessary due to an unforeseen issue where `t.type` caused modifications to the generated types (even when isPartial is true), leading to bugs when attempting to modify the Tanu object after that.